config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -45,6 +45,7 @@ presubmits:
           for binary in kubectl e2e.test ginkgo; do
             cp -f _output/local/go/bin/${binary} _output/dockerized/bin/linux/amd64/${binary};
           done;
+          make -C staging/src/k8s.io/kms/internal/plugins/base64/ setup-local-registry e2e-connect-registry;
```

Comment:
Having individual make targets will (1) make it hard to maintain (2) require an update to this file every time we need a new target or update an existing target.

Let's invoke a script here that'll perform all the necessary tasks for the CI. Maybe call it `test/e2e/testing-manifests/auth/encrypt/setup-cluster.sh`?